diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/LineLengthWarnText.kt

```diff
@@ -0,0 +1,208 @@
+package org.cqfn.diktat.ruleset.chapter3
+
+import com.pinterest.ktlint.core.LintError
+import org.cqfn.diktat.common.config.rules.RulesConfig
+import org.cqfn.diktat.ruleset.constants.Warnings.LONG_LINE
+import org.cqfn.diktat.ruleset.rules.DIKTAT_RULE_SET_ID
+import org.cqfn.diktat.ruleset.rules.LineLength
+import org.cqfn.diktat.util.lintMethod
+import org.junit.Test
+
+
+class LineLengthWarnText {
+
+    private val ruleId = "$DIKTAT_RULE_SET_ID:line-length"
+
+    private val rulesConfigListLineLength: List<RulesConfig> = listOf(
+            RulesConfig(LONG_LINE.name, true,
+                    mapOf("lineLength" to "155"))
+    )
+
+    @Test
+    fun `check correct example`() {
+        lintMethod(LineLength(),
+                """
+                    |package org.cqfn.diktat.ruleset.chapter3
+                    |
+                    |import org.cqfn.diktat.ruleset.rules.LineLength
+                    |import org.cqfn.diktat.util.lintMethod
+                    |
+                    |/**
+                    |*https://www.google.com
```

Comment:
```suggestion
                    | * https://www.google.com
```
Ususally (and in our code style too) asterisks in KDocs are followed by spaces. Could you please change your test cases and fix logic to correctly handle it?